src/ReflectionType.php

```diff
@@ -85,7 +85,7 @@ public static function convertToDisplayType(\ReflectionType $type)
             'int'  => 'integer',
             'bool' => 'boolean'
         ];
-        $displayType = $type->type;
+        $displayType = (string) $type;
```

Comment:
Are you sure, that according to coding standard there should be space between type name and variable?